schema/descriptor_test.go

```diff
@@ -24,7 +24,7 @@ import (
 func TestDescriptor(t *testing.T) {
 	for i, tt := range []struct {
 		descriptor string
-		fail     bool
+		fail       bool
```

Comment:
@stevvooe Can you please reopen this PR? I will delete the wrong changes, only to retain this one format changes.thanks.